internal/services/appservice/helpers/function_app_schema.go

```diff
@@ -540,6 +540,281 @@ func SiteConfigSchemaLinuxFunctionAppComputed() *pluginsdk.Schema {
 	}
 }
 
+type SiteConfigLinuxFunctionAppFlexConsumption struct {
+	AppCommandLine                string                      `tfschema:"app_command_line"`
+	ApiDefinition                 string                      `tfschema:"api_definition_url"`
+	ApiManagementConfigId         string                      `tfschema:"api_management_api_id"`
+	AppInsightsInstrumentationKey string                      `tfschema:"application_insights_key"` // App Insights Instrumentation Key
```

Comment:
Can we remove this comment here since it isn't really alluding to anything helpful (the name is already descriptive)
```suggestion
	AppInsightsInstrumentationKey string                      `tfschema:"application_insights_key"`
```